web/packages/core/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "extends": "@tsconfig/recommended/tsconfig.json",
     "compilerOptions": {
         "target": "es2017",
-        "module": "es2015",
+        "module": "es2020",
```

Comment:
Just from curiosity, why `es2020` is needed?

Reply:
For the `import()` function.